src/tema/FarenheitConversion/FarenheitToCelsius.java

```diff
@@ -0,0 +1,68 @@
+package tema.FarenheitConversion;
+
+import java.util.Scanner;
+
+/**
+ * Diplays the body temperature in celsius and if you are ill will inform the customer
+ */
+public class FarenheitToCelsius {
+    /**
+     * gets from user the temperature in fahrenheit, convets it to celsius and diplays the value to customer,
+     * if the temperature in celsius is greater then 37 it will also display that you are ill
+     * @param args
+     */
+    public static void main(String[] args) {
+        float farenheitTemp;
+        farenheitTemp = getNumber();
+        System.out.println("Your body temperature in Celsius degrees is " + convertToCelsius(farenheitTemp));
+        if (convertToCelsius(farenheitTemp) > 37){
+            System.out.println("You are ill!");
+        }
+    }
+
+    /**
+     * transforms the fahrenheit to celsius
+     * @param fahrenheitTemp value to be converted
+     * @return value is celsius
+     */
+    public static float convertToCelsius(float fahrenheitTemp){
```

Comment:
good method, good javadoc, good naming. You could have just returned directly instead of creating a variable celsius and returning the variable.